scripts/_common.sh

```diff
@@ -79,6 +87,7 @@ if [ -z "$RID" ]; then
     fi
 fi
 
+export REPOROOT=$(cd $COMMONDIR/.. && pwd)
```

Comment:
I was originally going to do this but was getting weird paths. Having said that, I was also messing with other things at the same time. Glad to have this fixed up! :+1: